src/SFA.DAS.EmployerAccounts.Web/ViewModels/Row1Panel1ViewModel.cs

```diff
@@ -0,0 +1,21 @@
+﻿using SFA.DAS.EmployerAccounts.Web.Extensions;
+using System;
+
+namespace SFA.DAS.EmployerAccounts.Web.ViewModels
+{
+    public class Row1Panel1ViewModel
+    {
```

Comment:
Does this need to be merged with the call to action view model now?

Reply:
Yes - but that is in the 1261 branch currently